CONTRIBUTING_GUIDE.adoc

```diff
@@ -37,12 +37,12 @@ An in-depth documentation on basic Git syntax and usage can be found on the http
 
 === Structure of our projects
 
-In total, there are three GitHub projects regarding OASP and devon:
+In total, there are three GitHub projects regarding DEVON and devon:
 
-* link:https://github.com/oasp-forge[*oasp-forge*]
+* link:https://github.com/devon-forge[*devon-forge*]
 +
 Repository used for work on the guide - Similar to the according devon repository link:https://github.com/devonfw/devon-guide/wiki[*devon-guide*]
-* link:https://github.com/oasp/[*oasp*]
+* link:https://github.com/devon/[*devon*]
```

Comment:
The link should start with https://github.com/devonfw

Reply:
Ok, it is related with the next change. This OASP references should be removed as they will not exist anymore.